src/Tizen.NUI/src/internal/FrameBroker/FrameBrokerBase.cs

```diff
@@ -364,7 +364,7 @@ internal Renderer GetRenderer(FrameData data)
                     break;
             }
 
-            texture.Dispose();
+            texture?.Dispose();
```

Comment:
How about geometry and shader ? also don't they need null check?

Reply:
> How about geometry and shader ? also don't they need null check?

They have been New in 
            Geometry geometry = CreateQuadGeometry();
            Shader shader = CreateShader();

And in the Switch case, function return Null directly:
            switch (data.Type)
            {
                case FrameData.FrameType.RemoteSurfaceTbmSurface:
                    if (data.TbmSurface == null)
                    {
                        geometry.Dispose();
                        shader.Dispose();
                        return null;
                    }